recipes/bioservices/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "bioservices" %}
+{% set version = "1.7.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bioservices-{{ version }}.tar.gz
+  sha256: ad169fb2a1971daddd67507b81b07cb9654dfea7b76a5926d0913229f1e9d0fe
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - appdirs
+    - beautifulsoup4
+    - colorlog
+    - easydev >=0.9.36
+    - grequests
+    - lxml
+    - pandas
+    - python >=3.6
+    - requests
+    - requests_cache
+    - suds-jurko
+    - wrapt
+    - xmltodict
+
+test:
+  imports:
+    - bioservices
+    - bioservices.apps
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/cokelaer/bioservices
+  summary: Access to Biological Web Services from Python
+  license: GPL-3.0-only
```

Comment:
Is it only or does it support newer versions of the license as well?

Reply:
How could I check that? See https://github.com/cokelaer/bioservices/blob/master/LICENSE

Reply:
How about this?

https://github.com/cokelaer/bioservices/blob/a334adaf6ea6ea87687517bfab0b0c090b68fcca/LICENSE#L570-L594

Reply:
Thanks for the catch!